docker/basics/step8_remove/verify.sh

```diff
@@ -1,3 +1,29 @@
 #!/bin/bash
 
-#docker run -it --rm --name hello-world alpine:latest "echo Hello World"
\ No newline at end of file
+#docker run -it --rm --name hello-world alpine:latest "echo Hello World"
+
+if ! history | grep -q "docker ps -a"; then
```

Comment:
I think we should rather check for the content of `docker ps` and `docker images`.  check if the cleanup occurred.